packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -559,9 +561,9 @@ export class MgtPersonCard extends MgtTemplatedComponent {
    * @memberof MgtPersonCard
    */
   protected renderContactIcons(person?: IDynamicPerson): TemplateResult {
-    if (this.isExpanded) {
-      return;
-    }
+    // if (this.isExpanded) {
```

Comment:
Should this still be commented out? If so, perhaps it would be better to remove the dead code?